x-pack/test/api_integration/apis/cloud_security_posture/status/status_unprivileged.ts

```diff
@@ -194,14 +245,35 @@ export default function (providerContext: FtrProviderContext) {
           .auth(UNPRIVILEGED_USERNAME, 'changeme')
           .expect(200);
 
-        expect(res.kspm.status).to.be('unprivileged');
-        expect(res.cspm.status).to.be('unprivileged');
-        expect(res.vuln_mgmt.status).to.be('not-installed');
+        expect(res.kspm.status).to.eql(
+          'unprivileged',
+          `expected unprivileged but got ${res.kspm.status} instead`
+        );
+        expect(res.cspm.status).to.eql(
+          'unprivileged',
+          `expected unprivileged but got ${res.cspm.status} instead`
+        );
+        expect(res.vuln_mgmt.status).to.eql(
+          'not-installed',
+          `expected not-installed but got ${res.vuln_mgmt.status} instead`
+        );
 
-        expect(res.indicesDetails[0].status).to.be('unprivileged');
-        expect(res.indicesDetails[1].status).to.be('empty');
-        expect(res.indicesDetails[2].status).to.be('unprivileged');
-        expect(res.indicesDetails[3].status).to.be('empty');
+        expect(res.indicesDetails[0].status).to.eql(
```

Comment:
this assertion is not robust. It depends on the order of the result which makes this assertion fragile to changes.
I suggest to have an assertion that checks for specific index and it's status

something like:
```res.indicesDetails.find(idx => idx.name === '...')?.status === '...'```